### ISSUE TITLE
Implement a `fetch_raw` option, `only_supports_raw` instead of `only_supports_classic`

### ISSUE THREAD

**Issue by brookslogan** [cmu-delphi/epidatr#32]
The endpoints marked with `only_supports_classic = TRUE`, such as `meta`, were designed in the original Epidata interface that received JSON and parsed it into a nested list structure.  Changing `only_supports_classic` to `only_supports_json` or `only_supports_classic_and_json` prevents some strange `print` output on the nested data frames (and may allow an easier transition for those using the original Epidata, although there are probably not many/any active human users at this point).

**Comment by brookslogan** [comment]
`fetch_json` doesn't actually do what I expected.  It doesn't take the json and just convert it to an R object; it tries for a data frame.  And `fetch_classic` also tries to do the same.  So some of these old endpoints may not output anything very nice for either `fetch_classic` or `fetch_json`.  We may need something different for them.

**Comment by brookslogan** [comment]
Another test case where we want "raw" format rather than "classic":
```r
ec_old = Epidata$delphi("ec", 201501)
ec = delphi("ec","201501") %>% fetch_classic()
```

It looks like the difference is due to `content(res, 'parsed')` vs. `r <- httr::content(res, "text", encoding = "UTF-8")` then `m <- jsonlite::fromJSON(r)`.

**Comment by dshemetov** [comment]
Closed by #99
